lib/chef/resource/execute.rb

```diff
@@ -65,7 +65,7 @@ def command(arg = nil)
                description: "The current working directory from which the command will be run."
 
       property :environment, Hash,
-               description: "Specify a Hash of environment variables to be set."
+               description: "A Hash of environment variables in the form of ({'ENV_VARIABLE' => 'VALUE'}). (These variables must exist for a command to be run successfully.)"
```

Comment:
The phrasing on this makes it seem a little like this is a required property. What about `If specified, these variables must exist...`

Reply:
That is super odd. This is from our docs site, but it really doesn't make sense.